x-pack/test/security_solution_cypress/cypress/tasks/create_new_rule.ts

```diff
@@ -125,26 +125,33 @@ import { BACK_TO_RULES_TABLE } from '../screens/rule_details';
 import { waitForAlerts } from './alerts';
 import { refreshPage } from './security_header';
 import { EMPTY_ALERT_TABLE } from '../screens/alerts';
+import { PAGE_CONTENT_SPINNER } from '../screens/common/page';
 
 export const createAndEnableRule = () => {
-  cy.get(CREATE_AND_ENABLE_BTN).click({ force: true });
+  cy.get(CREATE_AND_ENABLE_BTN).click();
   cy.get(CREATE_AND_ENABLE_BTN).should('not.exist');
-  cy.get(BACK_TO_RULES_TABLE).click({ force: true });
-  cy.get(BACK_TO_RULES_TABLE).should('not.exist');
+
+  // Wait for page to be loaded before performing any actions
+  // Serverless breadcrumbs don't work until the page is loaded
+  cy.get(PAGE_CONTENT_SPINNER).should('be.visible');
+  cy.get(PAGE_CONTENT_SPINNER).should('not.exist');
+
+  cy.get(BACK_TO_RULES_TABLE).click();
+  cy.get(BACK_TO_RULES_TABLE).should('have.attr', 'data-test-subj').and('match', /last/);
```

Comment:
I don't think this should be in the `createAndEnableRule` function. Some tests might need to do something on the Rule Details page, or use the API for assertions, so navigating to the Rule Management page by default is an extra step that should be done only in those tests that need this.

I understand that this could be out of the scope of this PR, so maybe we could create a ticket to track this.

Reply:
Right. I also had a doubt if it should be here but didn't want to make too much changes and create conflicts with my another [PR](https://github.com/elastic/kibana/pull/166201). The best we can do now is to add a helper function named like `goBackToRulesTableViaBreadcrumbs()` and move that two lines of code inside then add it after each `createAndEnableRule()` invocation. But it will create conflicts with https://github.com/elastic/kibana/pull/166501. So it 's better to merge back https://github.com/elastic/kibana/pull/166501 first.